deploy/helm/sumologic/values.yaml

```diff
@@ -2381,6 +2469,180 @@ otelagent:
+          endpoint: ${SUMO_ENDPOINT_DEFAULT_METRICS_SOURCE}
+          source_host: '%{host}'
+          metadata_attributes:
+            - k8s.*
+            - pod_.*
+            - namespace_.*
+
+            - Deployment
+            - Namespace
+            - container
+            - host
+            - node
+            - pod
+      processors:
+        resource:
+          attributes:
+            - key: k8s.namespace.name
+              from_attribute: namespace
+              action: upsert
+            - key: k8s.pod.name
+              from_attribute: pod
+              action: upsert
+        ## The memory_limiter processor is used to prevent out of memory situations on the collector.
+        memory_limiter:
+          ## check_interval is the time between measurements of memory usage for the
+          ## purposes of avoiding going over the limits. Defaults to zero, so no
+          ## checks will be performed. Values below 1 second are not recommended since
+          ## it can result in unnecessary CPU consumption.
+          check_interval: 5s
+
+          ## Maximum amount of memory, in MiB, targeted to be allocated by the process heap.
+          ## Note that typically the total memory usage of process will be about 50MiB higher
+          ## than this value.
+          limit_mib: 1900
+
+        ## The batch processor accepts spans and places them into batches grouped by node and resource
+        batch:
+          ## Number of spans after which a batch will be sent regardless of time
+          send_batch_size: 256
+          ## Time duration after which a batch will be sent regardless of size
+          timeout: 5s
+        k8s_tagger:
+          ## Has to be false to enrich metadata
+          passthrough: false
+          owner_lookup_enabled: true  # To enable fetching additional metadata using `owner` relationship
+          extract:
+            metadata:
+              ## extract the following well-known metadata fields
+              - containerId
+              - containerName
+              - clusterName
+              - daemonSetName
+              - deploymentName
+              - hostName
+              - namespace
+              - nodeName
+              - podId
+              - podName
+              - replicaSetName
+              - serviceName
+              - statefulSetName
+            annotations:
+              - tag_name: "pod_annotations_%s"
+                key: "*"
+            namespace_labels:
+              - tag_name: "namespace_labels_%s"
+                key: "*"
+            labels:
+              - tag_name: "pod_labels_%s"
+                key: "*"
+          pod_association:
+            - from: build_hostname
+        ## Source processor adds Sumo Logic related metadata
+        source:
+          collector: '{{ .Values.sumologic.collectorName | default .Values.sumologic.clusterName | quote }}'
+          source_name: '{{ .Values.fluentd.logs.containers.sourceName | quote }}'
```

Comment:
should these metadata be set to `.Values.fluentd.logs`?

Reply:
not sure, need to check what fluentd sets for `sourceName`